keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -327,9 +356,9 @@ No changes needed
 ###### How can this feature be enabled / disabled in a live cluster?
 
 - [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: `TopologyManagerPolicyOptions`
+  - Feature gate name: `TopologyManagerPolicyAlphaOptions`
```

Comment:
We should add all the feature gates here:  `TopologyManagerPolicyOptions`, `TopologyManagerPolicyAlphaOptions` and `TopologyManagerPolicyBetaOptions`

Reply:
Fixed.